exasol_integration_test_docker_environment/testing/exaslct_docker_test_environment.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+from typing import Optional
+
+from exasol_integration_test_docker_environment.lib.data.environment_info import EnvironmentInfo
+from exasol_integration_test_docker_environment.testing.utils import remove_docker_container, remove_docker_volumes
+
+
+class ExaslctDockerTestEnvironment:
```

Comment:
Good idea to extract. I also get the feeling, that we might need not only for test, maybe we need to move it soon to lib for the new API